VATSpy.dat

```diff
@@ -18157,6 +18157,7 @@ MDCS|Santo Domingo||MDCS
 MHCC|Cenamer||MHCC
 MKJK|Kingston||MKJK
 MMEX|Mexico|MEX|MMEX
+MMFR|Mexico|MEX|MMFR

```

Comment:
This will not work.
MEX should only be on one of these.

However if you do not use MEX_CTR // MEX_FSS as logons, they can just be removed all together.

Reply:
Removed MEX from both as they are not used.